### PR TITLE
Updating READMEs

### PR DESCRIPTION
**Pull request by autodidaddict**
Signed-off-by: Kevin Hoffman <autodidaddict@users.noreply.github.com>

### CODE REVIEW

**Comment by stevelr** [comment]
that's the last time i try to edit something in the github ui

**Review by stevelr — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+* The Host Core
+* Dashboard Web UI
+
+## Host Core
+The **host core** consists of all of the "headless" (no UI) functional components of the system. This OTP application and its contained supervision tree represent the _core_ of the wasmCloud OTP host runtime.
+
+You can find the [host core](./host_core/README.md) in this github repository.
+
+## Dashboard Web UI
+The dashboard web UI (often colloquially referred to as the _washboard_) is a **Phoenix** application that fits snugly atop the host core, providing real-time web access to a variety of information, telemetry, and insight while also exposing a graphical interface to controlling the host and portions of the lattice.
```
Comment (brooksmtownsend) [inline README.md:19]: I think a note here, similar to the above, saying `you can find the dashboard under [wasmcloud host] (link) in this repository` would be helpful as it's not named dashboard

File: host_core/README.md [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
+If you have a functioning Elixir development environment that includes NATS, then you can simply git pull this entire repository, `cd` into the `host_core` directory, and run `iex -S mix` to launch the application with an active `iex` console.
 
-```elixir
-def deps do
-  [
-    {:host_core, "~> 0.1.0"}
-  ]
-end
-```
+If instead you prefer to work from the production release, then consult our [installation]() guide for the exact instructions.
```
Comment (brooksmtownsend) [inline host_core/README.md:10]: Installation should go to documentation here

**Review by brooksmtownsend — CHANGES_REQUESTED** [review]
A couple small changes, otherwise looks good. Perhaps in the future we can look into getting code coverage, latest release, etc badges in this repo

**Review by brooksmtownsend — APPROVED** [review]

**Pull request merged**